digital_comms/ccam.py

```diff
@@ -98,13 +160,35 @@ def population_density(self):
             return self.population / total_area
 
     def add_pcd_sector(self, pcd_sector):
+        """Add a postcode sector to the local area district.
+
+        Arguments
+        ---------
+        pcd_sector: PostcodeSector
+            Representation of a postcode sector that needs to be
+            added to the local area district
+        """
         self._pcd_sectors[pcd_sector.id] = pcd_sector
 
     def add_asset(self, asset):
+        """Add an asset to postcode sector to the local area district
+
+        Arguments
+        ---------
+        asset: TODO
+            TODO
+        """
         pcd_sector_id = asset.pcd_sector
         self._pcd_sectors[pcd_sector_id].add_asset(asset)
 
     def system(self):
+        """TODO
```

Comment:
I think this just populates a dict with all of the existing assets, which in total represent the 'system'

Reply:
Done